torch/_dynamo/debug_utils.py

```diff
@@ -962,33 +958,43 @@ def dynamo_accuracy_minifier_backend(gm, example_inputs, compiler_name):
 
     from torch._dynamo.optimizations.backends import BACKENDS
 
+    # (1) Get the backend
     if compiler_name == "inductor":
         from torch._inductor.compile_fx import compile_fx
 
         compiler_fn = compile_fx
     else:
+        assert (
+            compiler_name in BACKENDS
+        ), f"Unknown compiler name {compiler_name} provided."
         compiler_fn = BACKENDS[compiler_name]
 
-    # Set the eval mode to remove randomness.
-    gm.eval()
+    # (2) Compile the graph
+    compiled_gm = compiler_fn(copy.deepcopy(gm), clone_inputs(example_inputs))
 
-    # Check Accuracy
-    if backend_accuracy_fails(gm, example_inputs, compiler_fn):
```

Comment:
What happened to this call?

Reply:
`if same_two_models` this call used to wrap it, we dont need the wrapper.

`pytest test/dynamo/test_minifier.py test/inductor/test_minifier.py` tests this :)

Reply:
So, the `backend_accuracy_fails` is inlined now. Does this mean there are two copies of the code? Should we delete the old one?

Reply:
We can collapse them together with a little modification, its used elsewhere